rest-service/manager_rest/upload_manager.py

```diff
@@ -723,7 +723,12 @@ def _load_plugin_package_json(wagon_source):
         #     raise manager_exceptions.InvalidPluginError(
         #         'the provided wagon can not be read.')
 
-        return wagon.show(wagon_source)
+        try:
+            return wagon.show(wagon_source)
+        except wagon.WagonError as e:
+            raise manager_exceptions.InvalidPluginError(
+                'The provided wagon archive can not be read.\n'
+                + e.message)
```

Comment:
We usually use string formatting: 
'The provided wagon archive can not be read. {0} '.format(e.message)